### PR TITLE
open-cluster-management/backlog#7439 Add clusterclaims

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
Hi @elgnay. Here is a draft of the cluster claims documentation. Let me know if you have any updates. Thank you!

File: manage_cluster/list_clusterclaim.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[#list-clusterclaims]
+= List existing ClusterClaims
+
+You can use the `kubectl` command to list the ClusterClaims that apply to your managed cluster. This is helpful when you want to compare your ClusterClaim to an error message. 
+
+*Note:* Make sure you have `list` permission on resource `clusterclaims.cluster.open-cluster-management.io`.
+
+Run the following command to list all existing ClusterClaims on the managed cluster:
```
Comment (dockerymick) [inline manage_cluster/list_clusterclaim.adoc]: ...existing ClusterClaims on the managed cluster 

to 

..list all existing ClusterClaims that are on the managed cluster.

File: manage_cluster/custom_clusterclaims.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[#create-custom-clusterclaims]
+= Create custom ClusterClaims
+
+You can create ClusterClaims with custom names on a managed cluster, which makes it easier to identify them. The custom ClusterClaims are synchronized with the status of the `ManagedCluster` resource on the hub cluster. The following content shows an example of a definition for a customized ClusterClaim:
```
Comment (dockerymick) [inline manage_cluster/custom_clusterclaims.adoc]: ClusterClaim should be monospaced

File: manage_cluster/clusterclaims.adoc [hunk 3]
```diff
@@ -0,0 +1,91 @@
+[#clusterclaims]
+= ClusterClaims
+
+A _ClusterClaim_ is a cluster-scoped custom resource definition (CRD) on a managed cluster. A ClusterClaim represents a piece of information that a managed cluster claims. The following example shows a claim that is identified in the YAML file:
```
Comment (dockerymick) [inline manage_cluster/clusterclaims.adoc]: Monospace for ClusterClaim

File: manage_cluster/clusterclaims.adoc [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  name: id.openshift.io
+spec:
+  value: 95f91f25-d7a2-4fc3-9237-2ef633d8451c
+----
+
+The following table shows the defined ClusterClaims that might be on a cluster that {product-title} manages:
+
+[cols="4"]
+|===
+| Claim Name | Reserved | Mutable | Description
```
Comment (dockerymick) [inline manage_cluster/clusterclaims.adoc]: Should the list of cluster claims be monospaced? I think we need to confirm how we want to handle **all** camel case (CamelCase) instances. Do we only want to use monospace if we aren't referencing the policy as a resource? For example:

* ClusterClaim resource (no monospace) 
vs
* `ClusterClaim` resource and `ClusterClaim`

File: manage_cluster/clusterclaims.adoc [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+| id.k8s.io | true | false | ClusterID defined in upstream proposal
+| kubeversion.open-cluster-management.io | true | true | Kubernetes version
+| platform.open-cluster-management.io | true | false | Platform the managed cluster is running on, like AWS, GCE, and Equinix Metal
+| product.open-cluster-management.io | true | false |  Product name, like OpenShift, Anthos, EKS and GKE
+| id.openshift.io | false | false | OpenShift external ID. It's available for OCP cluster only 
+| consoleurl.openshift.io | false | true | URL of management console. It's available for OCP cluster only
+| version.openshift.io | false | true | OCP version. It's available for OCP cluster only
+|===
+
+If any of the above claims is deleted or updated on managed cluster, it is restored or rolled back to a previous version automatically.
```
Comment (dockerymick) [inline manage_cluster/clusterclaims.adoc]: If any of the previously mentioned claims is deleted....

**Review by dockerymick — COMMENTED** [review]
left a few comments

File: manage_cluster/clusterclaims.adoc (hunk 4 above)
Comment (cadawson) [reply manage_cluster/clusterclaims.adoc]: Good point. I went ahead and made them monospaced.

**Pull request merged**